src/main/java/baza/trainee/controller/EventAdminController.java

```diff
@@ -0,0 +1,58 @@
+package baza.trainee.controller;
+
+import baza.trainee.domain.dto.event.EventPublication;
+import baza.trainee.domain.model.Event;
+import baza.trainee.service.EventService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("/admin/events")
+@RequiredArgsConstructor
+public class EventAdminController {
```

Comment:
No need additional words to path. Use REST aproach:

GET - /events - all events
GET - /events/{id} - event by Id
POST - /events - create new Event
PUT - /events/{id} - update by Id
DELETE - /events/{id} - delete by Id

Reply:
Done.